src/ff8/texture_packer.cpp

```diff
@@ -42,6 +42,8 @@ void TexturePacker::cleanVramTextureIds(const TextureInfos &texture)
 
 void TexturePacker::cleanTextures(ModdedTextureId previousTextureId, bool keepMods)
 {
+	ffnx_trace("%s: previousTextureId=%d keepMods=%d\n", __func__, previousTextureId, keepMods);
```

Comment:
Remember to put an `if` if it's intended to be kept.

Reply:
Yes, it is not intended to be kept